fe/fe-core/pom.xml

```diff
@@ -746,6 +746,22 @@ under the License.
             </exclusions>
         </dependency>
 
+        <!-- for fe recognize files stored on gcs -->
+        <dependency>
+            <groupId>com.google.cloud.bigdataoss</groupId>
+            <artifactId>gcs-connector</artifactId>
+            <version>hadoop3-2.1.4</version>
+            <classifier>shaded</classifier>
+            <!-- conflict with grpc-netty-shaded -->
```

Comment:
Considering that this jar was released in 2020, there are some CVEs of dependent components, have you considered upgrading the version?

Reply:
The latest version of the `gcs-connector` conflicts with the `grpc-netty-shaded`. You can upgrade to the latest version if you know how to resolve the conflict

Reply:
I think the reason of that  'gs://' is required maybe you use the gcs-connector?  I had the same problem when I used Huawei OBSClient before.

Reply:
> The latest version of the `gcs-connector` conflicts with the `grpc-netty-shaded`. You can upgrade to the latest version if you know how to resolve the conflict↳

If you can tease out their dependencies, and list version compatibility, then we can consider which version is more appropriate.
But one thing that needs to be explained, is the consideration of CVE is always the first and most important point.